shared/calculators/calculate_inverse_matrix_test.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+import * as tf from '@tensorflow/tfjs-core';
+// tslint:disable-next-line: no-imports-from-dist
+import {expectArraysClose} from '@tensorflow/tfjs-core/dist/test_util';
+
+import {arrayToMatrix4x4, calculateInverseMatrix, Matrix4x4, matrix4x4ToArray} from './calculate_inverse_matrix';
+
+describe('calculateInverseMatrix', () => {
+  const identity: Matrix4x4 =
+      [[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, 1]];
```

Comment:
We recently fixed how this calculator is working on close to zero values. I'd suggest you verify it for you implementation as well. I.e. for diagonal matrix [1e-5, 1e-5, 1, 1].

Reply:
Added